mm2src/lp_ordermatch.rs

```diff
@@ -1547,54 +1656,281 @@ enum OrderbookRequestingState {
     NotRequested { subscribed_at: u64 },
 }
 
+type H64 = [u8; 8];
+
+struct TrieDiff<Key, Value> {
+    delta: Vec<(Key, Option<Value>)>,
+    next_root: H64,
+}
+
+impl<Key: std::fmt::Debug, Value: std::fmt::Debug> std::fmt::Debug for TrieDiff<Key, Value> {
```

Comment:
I think this can be simplified by `#[derive(Debug)]`.
If Key or Value doesn't implement Debug, TrieDiff doesn't derive it either.

Reply:
I had some glitch with derive yesterday, will retry today and let you know :slightly_smiling_face:

Reply:
Well, the glitch turns out to be only in my mind :slightly_smiling_face:

Reply:
I think it could have happened for a reason that we are now unable to find out :)